analyses/cell-type-wilms-tumor-06/notebook/README.md

```diff
@@ -36,6 +36,19 @@ The next step in analysis is to identify tumor vs. normal cells.
 - `04_annotation_Across_Samples_exploration.html` is the output of the [`04_annotation_Across_Samples_exploration.Rmd`](../notebook/04_annotation_Across_Samples_exploration.Rmd) notebook. 
 In brief, we explored the label transfer results across all samples in the Wilms tumor dataset SCPCP000006 in order to identify a few samples that we can begin next analysis steps with.
 
+One way to evaluate the label transfer is to look at the mapping score for each label being transfered, which more or less correspond to the certainty for a label transfer to be _TRUE_. 
+We render the notebook with different thresholds for the mapping score and evaluate the impact of filtering out cells with a mapping score below 0.5, 0.75, 0.85 and 0.95.
```

Comment:
I wanted to point something out here from thee Azimuth docs: https://azimuth.hubmapconsortium.org/

I had been under the impression that the scores we were working with are what they are calling _prediction scores_, not mapping scores, but now I'm wondering whether I actually had a reason to think this. Do you know?

Reply:
Oh you are pointing out a good point. From what I read from the [Azimuth](https://azimuth.hubmapconsortium.org/), : 
both preduction and mapping scores exist and are the cell-level Metrics:

- _Prediction scores_: Cell prediction scores range from 0 to 1 and reflect the confidence associated with each annotation. Cells with high-confidence annotations (for example, prediction scores > 0.75) reflect predictions that are supported by mulitple consistent anchors. Prediction scores can be visualized on the Feature Plots tab, or downloaded on the Download Results tab. The prediction depends on the specific annotation for each cell. Therefore, if you are mapping cells at multiple levels of resolution (for example level 1/2/3 annotations in the Human PBMC reference), each level will be associated with a different prediction score.

- _Mapping scores_: This value from 0 to 1 reflects confidence that this cell is well represented by the reference. The “mapping.score” column is available to plot in the Feature Plots tab, and is provided in the download TSV file. The mapping score is independent of a specific annotation, is calculated using the MappingScore function in Seurat, and reflects how well the unique structure of a cell’s local neighborhood is preserved during reference mapping.

I am using the `predicted.score`, in fact I shouldn't refer it as `mapping`, you are right.

Reply:
I wasn't aware about the 2 metrics! thanks!

Reply:
Ah fantastic, I think the predicted.score is definitely want we want to be using!! So, let's just change the text to say prediction instead of mapping, but otherwise this is good!